### PR TITLE
feat: add sainsbury's to adopters.tsx

### PR DESCRIPTION
**Pull request by jfcurran**

### CODE REVIEW

File: src/components/adopters.tsx [hunk 1]
```diff
@@ -37,9 +38,9 @@ const adopters = [
     hiddenSm: false
   },
   {
-    title: 'Arduino',
-    image: arduino,
-    url: 'https://arduino.cc/',
+    title: 'Sainsburys',
```
Comment (vinckr) [inline src/components/adopters.tsx:41]:
```suggestion
    title: 'Sainsbury's',
```

**Review by vinckr — CHANGES_REQUESTED** [review]

File: src/components/adopters.tsx (hunk 1 above)
Comment (aeneasr) [reply src/components/adopters.tsx:41]: @vinckr escape ;)

Comment (jfcurran) [inline src/components/adopters.tsx:41]:
```suggestion
    title: 'Sainsbury's',
```

**Review by jfcurran — COMMENTED** [review]
apostrophe

File: src/components/adopters.tsx (hunk 1 above)
Comment (aeneasr) [reply src/components/adopters.tsx]: @jfcurran this will not compile. you need to escape this.

**Comment by vinckr** [comment]
LGTM
3 rows 😮

**Pull request merged**